front/pages/poke/[wId]/index.tsx

```diff
@@ -203,6 +220,30 @@ const WorkspacePage = ({
     }
   };
 
+  const onSlackbotToggle = async () => {
+    try {
+      const r = await fetch(
+        `/api/poke/workspaces/${workspace.sId}/data_sources/managed-slack/bot_enabled`,
+        {
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify({
+            botEnabled: !slackbotEnabled,
+          }),
+        }
+      );
+      if (!r.ok) {
+        throw new Error("Failed to toggle slackbot.");
+      }
+      router.reload();
```

Comment:
Just curious what is the UX with that router.reload()? Why do you need that?

Reply:
Does it mean that any other local state on the page is lost?

Reply:
(BTW, would be cool to talk to Dust from here).

Reply:
I mimicked the click on delete a datasource; I think there is no local state on the page
The reload is needed for the toggle status to change to the new value. Glad to know if there's a (quick to implement) better way